core/commands/log.py

```diff
@@ -21,13 +21,10 @@ class LogMixin(object):
     """
 
     def run(self, *args, file_path=None, branch=None):
-        self._file_path = file_path
-        self._branch = branch
-        sublime.set_timeout_async(self.run_async)
-
-    def run_async(self):
-        show_log_panel(self.commit_generator(), self.do_action)
+        sublime.set_timeout_async(lambda: self.run_async(file_path, branch), 0)
 
+    def run_async(self, file_path, branch):
+        show_log_panel(self.log_generator(file_path=file_path, branch=branch), self.do_action)
```

Comment:
Now  I added `**kwards` where it is in use. Should I add `file_path=None` instead on those locations? I don't think we will have any other arguments passed around here any time soon so manybe it is better to only take `file_path`?
https://github.com/divmain/GitSavvy/pull/661/commits/c52f85c639c24eea7cadeb623c40ded76b5a29fa